src/chmod.js

```diff
@@ -4,31 +4,31 @@ var path = require('path');
 
 var PERMS = (function (base) {
   return {
-    OTHER_EXEC  : base.EXEC,
-    OTHER_WRITE : base.WRITE,
-    OTHER_READ  : base.READ,
+    OTHER_EXEC: base.EXEC,
+    OTHER_WRITE: base.WRITE,
+    OTHER_READ: base.READ,
 
-    GROUP_EXEC  : base.EXEC  << 3,
-    GROUP_WRITE : base.WRITE << 3,
-    GROUP_READ  : base.READ << 3,
+    GROUP_EXEC: base.EXEC << 3,
+    GROUP_WRITE: base.WRITE << 3,
+    GROUP_READ: base.READ << 3,
 
-    OWNER_EXEC  : base.EXEC << 6,
-    OWNER_WRITE : base.WRITE << 6,
-    OWNER_READ  : base.READ << 6,
+    OWNER_EXEC: base.EXEC << 6,
+    OWNER_WRITE: base.WRITE << 6,
+    OWNER_READ: base.READ << 6,
 
     // Literal octal numbers are apparently not allowed in "strict" javascript.  Using parseInt is
     // the preferred way, else a jshint warning is thrown.
```

Comment:
Two things:

1. It might be good to note that they've been added back in ES6, for future reference.
2. Maybe change the reference to jshint to eslint?